ern-api-impl-gen/src/generators/ApiImplGen.js

```diff
@@ -31,11 +36,15 @@ export default class ApiImplGen {
 
     await this.downloadApiAndDependencies(apiDependency, paths.pluginsDownloadDirectory, reactNativeVersion)
 
+    const schemaJson = path.join(paths.pluginsDownloadDirectory, `node_modules`, apiDependency.scopedName, `schema.json`)
```

Comment:
Minor but let's use single quote instead of backtick when the string does not contain any placeholder (i.e just a string, not a template litteral). I already started doing that in some places.